### PR TITLE
Add a page to display benchmarks

### PR DESCRIPTION
**Pull request by AMoo-Miki**
Add a page to display benchmarks
* Simple `IFRAME` to an OSD dashboard
* The page is temporarily excluded from search engines

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by bbarani — DISMISSED** [review]

**Comment by krisfreedain** [comment]
This is amazing @AMoo-Miki @bbarani - thank you. 

I've asked @jhmcintyre to review some of the wording included at the top of the page to see if we need to be more descriptive -- I expect he'll be able to get time for that on Monday - thanks!

File: benchmarks/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            height: 12000px;
+        }
+    }
+</style>
+<p>
+    Welcome to OpenSearch benchmarking page. The performance tests are executed in regular cadence using the workloads mentioned below for recently released OpenSearch version along with upcoming OpenSearch versions. You can view the key performance metrics using the below dashboard.
+</p>
+<iframe
+  id="benchmark-dashboard"
+  src="https://benchmarks.opensearch.org/app/dashboards?security_tenant=global#/view/88071cb0-f118-11ed-aff5-859eb6ed880f?embed=true&show-time-filter=true&hide-filter-bar=true&&_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-2d,to:now))"
```
Comment (bbarani) [inline benchmarks/index.html:24]: Please update  `from:now-2d` to `from:now-7d`

**Review by bbarani — APPROVED** [review]

**Comment by krisfreedain** [comment]
@AMoo-Miki @bbarani -- are we ready on this one?

**Comment by bbarani** [comment]
> @AMoo-Miki @bbarani -- are we ready on this one?

I am good with merging this PR.

**Review by krisfreedain — APPROVED** [review]
LGTM - this is fantastic

**Pull request merged**